### ISSUE TITLE
tvOS support

### ISSUE THREAD

**Issue by honkmaster** [vtourraine/AcknowList#35]
Hi,

I was trying to use `AcknowList` with tvOS, similar to `VTAcknowledgementsViewController`. However, it does not work :):

```
[!] The platform of the target `redacted` (tvOS 11.0) is not compatible with `AcknowList (1.5)`, which does not support `tvos`.
```

Is this just a CocoaPods or a repository/source issue?

Best

**Comment by vtourraine** [comment]
Hello! No, I never got around to add tvOS support. But I just saw your branch and so... now I’m looking forward to your pull request :smile:

**Comment by honkmaster** [comment]
Check.

**Comment by honkmaster** [comment]
Quick question: The minimum OS version is 8.0, right? Thus, I would replace `UIFont.systemFont(ofSize: 12)` with `UIFont.preferredFont(forTextStyle: .footnote)`.

**Comment by honkmaster** [comment]
Could you please create a new release for Cocoapods?

**Comment by vtourraine** [comment]
@honkmaster Just released version 1.6 with tvOS support on CocoaPods. 🚀

**Comment by honkmaster** [comment]
🍻
